persistence-modules/hibernate-queries/pom.xml

```diff
@@ -49,9 +81,34 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <version>${jmh-generator.version}</version>
         </dependency>
-    </dependencies>
+        
+        <dependency>
```

Comment:
Do we really need these jaxb* dependencies?

Reply:
Unused jaxb* dependencies have been removed. The comment has been addressed.